README.md

```diff
@@ -25,13 +25,16 @@ See [the installation guide](https://model-checking.github.io/kani/install-guide
 Similar to testing, you write a harness, but with Kani you can check all possible values using `kani::any()`:
 
 ```rust
-use my_crate::{function_under_test, meets_specification};
+use my_crate::{function_under_test, meets_specification, precondition};
 
 #[kani::proof]
 fn check_my_property() {
    // Create a nondeterministic input
    let input = kani::any();
 
+   // Constrain it according to precondition
```

Comment:
Relying on precondition solely as a word could be too technical for the README imo. This sentence could be rephrased as -
"Restrict input to valid values required by a boolean - precondition"

Reply:
I think I prefer the current wording. You could add something like "the function's pre-condition".